### PR TITLE
Versioned2

### PR DESCRIPTION
**Pull request by scontain**
- added: ignore security advisories
- added: aepic leak tolerance

### CODE REVIEW

File: check_prerequisites.sh [hunk 1]
```diff
@@ -164,7 +164,7 @@ if ! [[ "$STATUS" == "\"HEALTHY\"" ]]
 then
     echo -e "${RED}It seems that CAS '$CAS' in namespace '$CAS_NAMESPACE' is not healthy: status is $STATUS${NC}"
     echo -e "- ${ORANGE}- You can install cas as follows: kubectl provision cas $CAS $CAS_NAMESPACE -v${NC}"
-    error_exit
+    # error_exit
```
Comment (scontainyourself) [inline check_prerequisites.sh:167]: Don't just comment out, if it should be removed. But why should it not error_exit here?

File: mesh.yaml.template [hunk 2]
```diff
@@ -12,7 +12,9 @@ cas:
 
 policy:
   namespace: $APP_NAMESPACE    # namespace on CAS instance `cas`
-  tolerate: "outdated-tcb, insecure-configuration,hyperthreading, insecure-igpu, software-hardening-needed"
+  tolerate: "outdated-tcb, insecure-configuration,hyperthreading, insecure-igpu, software-hardening-needed, aepic-leak"
+  ignore_advisories: "*"
+#  ignore_advisories: "INTEL-SA-00586, INTEL-SA-00614, INTEL-SA-00615, INTEL-SA-00730, INTEL-SA-00738, INTEL-SA-00767"
```
Comment (scontainyourself) [inline mesh.yaml.template:17]: Why not removing the commented line?

File: mesh.yaml.template [hunk 3]
```diff
@@ -12,7 +12,9 @@ cas:
 
 policy:
   namespace: $APP_NAMESPACE    # namespace on CAS instance `cas`
-  tolerate: "outdated-tcb, insecure-configuration,hyperthreading, insecure-igpu, software-hardening-needed"
+  tolerate: "outdated-tcb, insecure-configuration,hyperthreading, insecure-igpu, software-hardening-needed, aepic-leak"
+  ignore_advisories: "*"
```
Comment (scontainyourself) [inline mesh.yaml.template:16]: Do we need ignore_advisories?

File: service.yaml.template [hunk 4]
```diff
@@ -17,6 +17,8 @@ environment:
       value: "$$SCONE::password$$"  # get from CAS
     - name: RELEASE
       value: $RELEASE
+    - name: EXCLAIM
```
Comment (scontainyourself) [inline service.yaml.template:20]: Is this used somewhere? Why add it?

File: service.yaml.template [hunk 5]
```diff
@@ -26,6 +28,9 @@ secrets:
   - name: password
     kind: ascii
     size: 10
+  - name: xpassword
```
Comment (scontainyourself) [inline service.yaml.template:31]: Is this used somewhere? Why add it?

File: check_prerequisites.sh (hunk 1 above)
Comment (scontain) [reply check_prerequisites.sh:167]: Done

File: mesh.yaml.template (hunk 3 above)
Comment (scontain) [reply mesh.yaml.template:16]: Yes, to ensure that we can run even on outdated servers

File: service.yaml.template (hunk 4 above)
Comment (scontain) [reply service.yaml.template:20]: Extended the Python program to use this explicitly defined secret

**Comment by scontainyourself** [comment]
`SCONECTL_REPO=registry.scontain.com/cicd VERSION=5.8.0-rc.25 CAS_VERSION=5.8.0-rc.24 ./run.sh -i registry.scontain.com/cicd --release mesh-tutorial` works

**Review by scontainyourself — APPROVED** [review]

**Pull request merged**